kubernetes/README.md

```diff
@@ -245,30 +246,38 @@ Note that these are the recommended `requests` and `limits` values for both the
 <details>
   <summary>High Volume Deployment</summary>
 
-  ```yaml
-  requests:
-    cpu: 500m
-    memory: 512M
+```yaml
+requests:
+  cpu: 500m
+  memory: 512M
 
-  limits:
-    cpu: 1000m
-    memory: 1024M
-  ```
+limits:
+  cpu: 1000m
+  memory: 1024M
+```
 
 </details>
 
 <details>
   <summary>Very High Volume Deployment</summary>
 
-  ```yaml
-  requests:
-    cpu: 1000m
-    memory: 1024M
+The best practices for Kubernetes resource limits are:
+
+1.  Never set [CPU](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource) limits to avoid Kubernetes CPU throttling and allow for efficient use of available CPU.
+2.  Set [Memory](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource) limits same as requests to help avoiding OOM kills.
 
-  limits:
-    cpu: 2000m
-    memory: 2048M
-  ```
+This works well in the deployment scenario where the 1Password SCIM bridge is the only pod in the cluster, so that it has a priority of service and can consume all the resources in the cluster without affecting the performance of other pods.
+
+If you are deploying the SCIM bridge to a cluster with other production services, we recommend using the following resources depending on the size of your deployment.
+
+```yaml
+requests:
+  cpu: 1000m
+  memory: 1024M
+
+limits:
+  memory: 1024M
```

Comment:
Is there a reason we left off the CPU limit of `cpu: 2000m`?

Reply:
So as part of no CPU limit strategy we remove it from limits to in order to decrease occurrences of CPU throttling and make full use of available CPU.

Reply:
That reads better. Thanks. 😄  
Changes applied.